statchatui/src/Stats/Stats.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const Stats = ({ statData }) => {
+  const kills = statData?.total?.kills?.value;
+  return (
+    <div className="container">
+      {statData ? (
+        <div>
+          <h1>Stats:</h1>
+          <p>Kills: {kills}</p>
+        </div>
+      ) : (
+        <div>Loading...</div>
```

Comment:
Ideally we don't want to show "Loading" until after the user clicks Submit. We should only show it while we're waiting for the API to return data. This will make the app feel a lot more snappy. 

 Feel free to implement this in a follow-up PR (can you add this to your document?).

Reply:
i changed it to say stats not fetched yet, is this ok

Reply:
We still want to have a loading state of some kind - this is one of the MetaU project requirements!

"Stats not fetched" is fine before the user submits anything.

Reply:
ok will add to milestones